src/runtime/types.ts

```diff
@@ -405,6 +405,15 @@ export interface ModuleOptions {
    * @default false
    */
   disableServerSideAuth?: boolean;
+  /**
+   * The environment key that holds the origin of the application. This is used to determine the full URL of the application in production.
```

Comment:
The key doesn't actually hold anything - it only points to the env variable:
```suggestion
   * The name of the environment variable that holds the origin of the application. This is used to determine the full URL of the application in production.
```